frontend/pages/index.jsx

```diff
@@ -1,12 +1,44 @@
 import React from 'react';
+import fetch from 'isomorphic-unfetch';
 import Layout from 'components/Layout';
-import GlobalStyles from '../styles/GlobalStyles';
+import GlobalStyles from 'styles/GlobalStyles';
+import OurGames from 'components/OurGames';
+import PropTypes from 'prop-types';
 
-const Index = () => (
+const Index = ({ games }) => (
   <>
     <GlobalStyles />
-    <Layout />
+    <Layout hasBear>
+      <OurGames games={games.projects} />
+    </Layout>
   </>
 );
 
+Index.getInitialProps = async function () {
+  const res = await fetch('http://localhost:3000/api/projects/');
```

Comment:
We shouldn't make requests to certain urls as on production we won't have access to localhost and instead it will be hosted on a different domain

```suggestion
  const res = await fetch('/api/projects/');
```